src/Apps/Order/Routes/__tests__/Payment.test.tsx

```diff
@@ -504,8 +513,9 @@ describe("Payment", () => {
       paymentRoute.find("Button").simulate("click")
       expect(stripeMock.createToken).toBeCalled()
     })
-    it("allows a missing state/province if the selected country is not US or Canada", () => {
-      const paymentRoute = mount(<PaymentRoute {...testProps} />)
+
+    it.only("allows a missing state/province if the selected country is not US or Canada", () => {
```

Comment:
@alloy - would you mind pulling down this branch and seeing if you can fix the error below? I'm a bit stumped. 

```
yarn jest src/Apps/Order/Routes/__tests__/Payment.test.tsx --watch --verbose false
```

There are quite a few strange test errors throughout and I suspect if the below is fixable then the rest will mostly resolve (after being wrapped in `MockBoot`).